test/fromZigbee.test.js

```diff
@@ -2,7 +2,7 @@ const legacy = require('../src/lib/legacy');
 
 jest.mock('fs');
 const fs = require('fs');
-const {fromZigbee} = require('../index');
+const {fromZigbee} = require('../src/index');
```

Comment:
Test kept failing locally on this report. This fixed it but might be wrong.

```bash
 PASS  test/ota.test.ts (14.571 s)
 PASS  test/lumi.test.js
 PASS  test/philips.test.js
 PASS  test/color.test.js
 PASS  test/otaCommon.test.ts
 PASS  test/utils.test.js (16.371 s)
 FAIL  test/fromZigbee.test.js
  ● Test suite failed to run

    Cannot find module '../index' from 'test/fromZigbee.test.js'

      3 | jest.mock('fs');
      4 | const fs = require('fs');
    > 5 | const {fromZigbee} = require('../index');
        |                      ^
      6 |
      7 | describe('converters/fromZigbee', () => {
      8 |     describe('tuya', () => {

      at Resolver._throwModNotFoundError (node_modules/jest-resolve/build/resolver.js:427:11)
      at Object.<anonymous> (test/fromZigbee.test.js:5:22)

 PASS  test/generateDefinition.test.ts (16.496 s)
 PASS  test/sonoff.test.ts (16.483 s)
 PASS  test/modernExtend.test.ts (16.573 s)
 PASS  test/index.test.js (18.299 s)

Test Suites: 1 failed, 10 passed, 11 total
Tests:       31 skipped, 223 passed, 254 total
Snapshots:   0 total
Time:        18.701 s, estimated 20 s
```

Reply:
`src/index` is correct indeed, thanks for fixing